themes/default/content/blog/top-5-things-for-azure-devs-static-websites/index.md

```diff
@@ -0,0 +1,493 @@
+
+    // Web endpoint to the website
+    export const staticEndpoint = storageAccount.primaryEndpoints.web;
+
+    {{% /choosable %}}
+{{< /chooser >}}
+
+The complete examples for [C#](https://github.com/pulumi/examples/tree/master/azure-cs-static-website),[Go](https://github.com/pulumi/examples/blob/master/azure-go-static-website/),[Python](https://github.com/pulumi/examples/blob/master/azure-py-static-website/), and [Typescript](https://github.com/pulumi/examples/blob/master/azure-ts-static-website/index.ts) are on Github.
+
+## Azure Static Web Apps
+
+Azure Static Web Apps (SWA) builds and deploys web apps from a code repository. Azure works directly with Githu or Azure Devops to watch a branch on your repository. When you push a commit or accept a pull request, your app is built and deployed to Azure. When you build web apps with SWA, your static assets are deployed to a CDN that serves files from geographically distributed points.
+
+Static web apps can be built with popular frameworks that don't require server-side rendering, such as React, Angular, and Vue. Web applications typically include HTML, JavaScript, images, and CSS. In addition to these assets, you can create API endpoints hosted in an Azure Function.
+
+The key thing to remember with SWAs is that you build the application from a repository.
+
+Let's jump into Azure Static Web Apps!
+
+### Build and deploy
+
+We'll use Visual Studio Code's [Azure Static Web App Extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestaticwebapps) and a [Github](https://github.com/) account.
+
+**Step 1**: Install the [Azure Web Static Web Apps](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestaticwebapps) extension. Follow the instructions to authorize Visual Studio Code to access your Github account.
+
+**Step 2**: Create a web application
+
+For convenience, we can use the Azure template for generating web applications. Click on this [link](https://github.com/staticwebdev/vanilla-basic/generate) to generate a basic React application.
+
+![Generate app from template](./react-basic.png)
+
+Clone the repository to your computer.
+
+```bash
+$ git clone https://github.com/spara/bookish-doodle.git
+```
+
+**Step 3**: Create a static web app with Visual Studio Code
+
+Open your repository with Visual Studio Code and activate the [Azure extension](https://code.visualstudio.com/docs/azure/extensions). On `APP SERVICE`item, click on the `+` to create a web app.
+
+![Create web app](./static_web_app_vscode.png)
+
+Follow the prompts:
+
+- Create a unique name.
+
+![Enter a name](./name.png)
+
+- Select project structure.
+
+![Select project structure](./react.png)
+
+- Select application directory.
+
+![Select project directory](./project_location.png)
+
+- Select build directory.
+
+![Select build directory](./build_directory.png)
+
+When the Visual Studio Code completes configuring your application, you will be prompted to deploy.
```

Comment:
More of a suggestion; folks searching for `VS Code` will get this page as a search result.

```suggestion
When VS Code completes configuring your application, you will be prompted to deploy.
```